core/src/main/java/com/linecorp/armeria/client/AbstractClientFactory.java

```diff
@@ -48,6 +50,32 @@
         return newClient(uri, clientType, ClientOptions.of(options));
     }
 
+    @Override
+    public final <T> T newClient(Scheme scheme, Endpoint endpoint, Class<T> clientType,
```

Comment:
It's not a part of this PR, but don't we need to do `null`-check for `clientType` as well? Would you please add it? :-)

Reply:
Sure~ I added null check for `clientType` at here.

Reply:
Thank you! 👍